packages/commonwealth/server/routes/contracts/createContract.ts

```diff
@@ -106,6 +135,11 @@ const createContract = async (
   });
 
   if (oldContract && oldContract.address === address) {
```

Comment:
@ianrowan is the new logic below valid? Before this change, if the contract address already existed in the `Contracts` table, we would return the contract directly and in 99% of cases this leads to a `missing abi_id` error from a GET request to `/contract/template`. The addition below finds or creates the ABI so that it can be linked to the old contract.